.github/workflows/ci-build.yaml

```diff
@@ -124,6 +124,7 @@ jobs:
 
 
   documentation:
+    name: Documentation - test code snippets
```

Comment:
For this, will copybara need to be changed?

Reply:
No. Our copybara config does not actually check the documentation build (either of them).